src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -291,7 +292,82 @@ public void RecalcEngine_Symbol_CultureInfo2()
             Assert.Equal("2,01", (expr2.Eval(fr_Symbols) as StringValue).Value);
         }
 
-        // Verify that an egine with a specific culture can evaluate an invariant formula
+        // Verify if text is transformed using the correct culture info (PowerFxConfig and Symbols)
+        [Fact]
+        public void RecalcEngine_Symbol_CultureInfo3()
+        {
+            var config = new PowerFxConfig(CultureInfo.InvariantCulture);
+            var engine = new RecalcEngine(config);
+
+            var tr_symbols = new SymbolValues();
+
+            tr_symbols.AddService(new CultureInfo("tr-TR"));
+
+            var textExpression = "Upper(\"indigo\")";
+            var datetimeExpression = "Text(DateTimeValue(\"20/1/2014 1:50:24.765 PM\", \"tr-TR\"))";
```

Comment:
>20/1/2014 1:50:24.765 PM [](http://example.com/codeflow?start=59&length=24)

Shouldn't we try with something that wouldn't work in English like "Perşembe 06 Ekim 2022 14:19:06"